docs/turbo-native-modules.md

```diff
@@ -20,7 +20,7 @@ The basic steps are:
 Lets work through each of these steps by building an example Turbo Native Module. The rest of this guide assume that you have created your application running the command:
 
 ```shell
-npx @react-native-community/cli@latest init TurboModuleExample --version 0.76.0
+npx @react-native-community/cli@latest init TurboModuleExample --version 0.81.0
```

Comment:
It might be better to specify versions for `@react-native-community/cli` instead of using `latest`, since each cli has compatible `react-native` versions.
https://github.com/react-native-community/cli?tab=readme-ov-file#compatibility